spinalcordtoolbox/scripts/sct_process_segmentation.py

```diff
@@ -320,17 +352,30 @@ def main(argv=None):
                                          angle_correction=angle_correction,
                                          param_centerline=param_centerline,
                                          verbose=verbose)
+    if fname_pmj is not None:
+        mask, pmj_slices = get_slices_for_pmj_distance(fname_segmentation, fname_pmj,
+                                                       distance_from_pmj, extent_mask,
+                                                       param_centerline=param_centerline,
+                                                       verbose=verbose)
+
+        fname_mask_out = add_suffix(arguments.i, '_mask_csa')
+        mask.save(fname_mask_out)
+        pmj_slices = [parse_num_list(pmj_slices[0]), pmj_slices[-1]]
```

Comment:
Do you think there is a possibility to instead update the variable `slices`, so that the API `aggregate_slicewise.py` would not need to be updated at all?

Reply:
I though about this, it was my first idea, but we cannot also output the distance from PMJ that the user specified this way, I think that is necessary to have in the .csv file.
Also, if we want to add the possibility to input multiple distances (so various range of slices) it wouldn't work with `slices ` (from my understanding) (if this is a feature we want of course)

Reply:
> I thought about this, it was my first idea, but we cannot also output the distance from PMJ that the user specified this way, I think that is necessary to have in the .csv file.

If the goal is to just populate the CSV file then this single argument could be passed to the API

> Also, if we want to add the possibility to input multiple distances (so various range of slices) it wouldn't work with slices (from my understanding) if this is a feature we want of course

I don't think we will be needing this feature.

Reply:
> If the goal is to just populate the CSV file then this single argument could be passed to the API

okay! done in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3429/commits/0696bb0abe42ee8b9a9bf72ecb87a467092ccd7e 😊